js/typeof.js

```diff
@@ -1,7 +1,11 @@
 console.log("ioan".constructor + "<br>" +
     (9.99).constructor + "<br>" +
     false.constructor + "<br>" +
+<<<<<<< HEAD
     [1, , 6, 4].constructor + "<br>" +
+=======
+    [1, , 4, 4].constructor + "<br>" +
+>>>>>>> 108cdea1a86ff19d5af6624c425c42b4272d49de
```

Comment:
This are conflicts, you need to resolve them. HEAD section is what you have on git server and second part is what you have on local.

You will need to keep just one section and delete this chars: "<<<<<<< HEAD" "=======" ">>>>>>> 108cdea1a86ff19d5af6624c425c42b4272d49de"

After that you need to execute 
**git add**
**git merge --continue**
**git push ...**